CHANGELOG.rst

```diff
@@ -1,6 +1,6 @@
-=======
```

Comment:
the new pre-commit check for merge conflict doesn't like the =======.... it assumes there's a conflict there

Reply:
Check the sphinx output on this one.  Sphinx is a little picky about the number of chars matching the number of chars in the string the formatting is being applied to.   It is not an error to sphinx, but it usually generates a warning.

Reply:
I have checked the rendering of the Queries example and it works well.
The changelog is generated only on the repo page, so I cannot see it directly. However, I have tried different RST viewers and it gets rendered correctly, so I believe we should be fine

Reply:
![image](https://github.com/user-attachments/assets/4651af0c-354a-4a45-8359-c328ee30fb51)